src/java/org/apache/cassandra/config/Config.java

```diff
@@ -596,6 +596,7 @@ public MemtableOptions()
     public volatile boolean use_statements_enabled = true;
 
     public boolean accord_transactions_enabled = false;
+    public long accord_scheduler_delay_in_ms = 200L;
```

Comment:
This could also be named `accord_progress_log_scheduler_delay_in_ms` but I felt it to verbose compared to the current one. I am open to suggestions on it.

Reply:
@kamalesh0406 @dcapwell  is not this "wrong" way of doing it? There are no "...in_ms" properties anymore, everything is based on `DurationSpec` when it comes to time.